examples/use-as-ts-module/tsconfig.json

```diff
@@ -0,0 +1,44 @@
+{
+  "compilerOptions": {
+    "baseUrl": ".",
+    "paths": {
+      "@textlint/*": [
+        "./packages/@textlint/*/src"
+      ],
+      "*": [
+        "./packages/*/src"
+      ]
+    },
```

Comment:
Probably, extends `tsconfig.base.json` resolve this issue?

CI error is sImilar to #414 ?